products/green_fast_track/models/_sources.yml

```diff
@@ -224,6 +238,27 @@ sources:
 
       - name: dcp_boroboundaries_wi
 
+      - name: dpr_schoolyard_to_playgrounds
+        columns:
+          - name: gispropnum
+            tests:
+              - not_null
+          - name: location
+            tests:
+              - not_null
+          - name: wkb_geometry
+            tests:
+              - not_null
+        tests:
+          - dbt_utils.unique_combination_of_columns:
+              name: dpr_schoolyard_to_playgrounds_compound_key
+              combination_of_columns:
+                - gispropnum
+                - location
+                - wkb_geometry
+              config:
+                severity: warn
```

Comment:
since later we do `gispropnum || '-' || location AS variable_id`, could this just be
```yaml
name: dpr_schoolyard_to_playgrounds_compound_key
  combination_of_columns:
    - gispropnum
    - location
```

Reply:
done!